database/factories/ModelFactory.php

```diff
@@ -21,3 +26,72 @@
         'visit_token' => $faker->asciify('********************'),
     ];
 });
+
+$factory->define(App\DuesPackage::class, function (Faker\Generator $faker) {
+    return [
+        'id' => $faker->numberBetween($min = 0, $max = 200),
+        'name' => $faker->word,
+        'eligible_for_shirt' => $faker->numberBetween($min = 0, $max = 1),
+        'eligible_for_polo' => $faker->numberBetween($min = 0, $max = 1),
+        'effective_start' => $faker->dateTime(),
+        'effective_end' => $faker->dateTime(),
+        'cost' => (string) $faker->randomFloat($nbMaxDecimals = 2, $min = 0, $max = 1000),
+    ];
+});
+
+$factory->define(App\DuesTransaction::class, function (Faker\Generator $faker) {
+    return [
+        'id' => $faker->numberBetween($min = 0, $max = 200),
+        'swag_shirt_provided' => $faker->optional()->dateTime,
+        'swag_shirt_providedBy' => User::first() != null ? User::first()->id : null,
+        'swag_polo_provided' => $faker->optional()->dateTime,
+        'swag_polo_providedBy' => User::first() != null ? User::first()->id : null,
+        'dues_package_id' => $faker->randomElement($array = idMap(DuesPackage::all())),
+        'user_id' => $faker->randomElement($array = idMap(User::all())),
+    ];
+});
+
+$factory->define(App\User::class, function (Faker\Generator $faker) {
+    $lastName = $faker->lastName;
+    $uid = $faker->bothify('?'.lcfirst($lastName).'##');
+
+    return [
+        'id' => $faker->numberBetween($min = 0, $max = 200),
+        'uid' => $uid,
+        'gtid' => $faker->numerify('#########'),
+        'slack_id' => null,
+        'gt_email' => $uid.'@gatech.edu',
+        'personal_email' => $faker->safeEmail,
+        'first_name' => $faker->firstName,
+        'middle_name' => $faker->optional()->lastName,
+        'last_name' => $lastName,
+        'preferred_name' => $faker->firstName,
```

Comment:
You can probably leave this out, or make it optional. No need to set preferred to the first name if they're the same.